cmd/dcrdata/public/scss/charts.scss

```diff
@@ -418,6 +418,7 @@ body.darkBG .chartview .dygraph-y2label {
   margin: 0.3rem auto;
   border: black;
   cursor: pointer;
+  width: unset;
```

Comment:
Oh, one thing I'd like to request is a bump to the app.bundle.js cache buster at https://github.com/decred/dcrdata/blob/5a46be532ea72ff7fac403bd091c350b55e3dd25/cmd/dcrdata/views/extras.tmpl#L189